index.js

```diff
@@ -73,3 +89,14 @@ function isGlobby(str) {
   }
   return isGlob(str);
 }
+
+function getWinDriveOrUncVolume(fp) {
+  if (/^([a-zA-Z]:|\\\\)/.test(fp)) {
+    var root = path.win32.parse(fp).root;
+    if (path.win32.isAbsolute(fp)) {
+      root = root.slice(0, -1);  // Strip last path separator
+    }
+    return root;
+  }
+  return '';
```

Comment:
Instead of using `path.win32` APIs, can we instead iterate the string and process the root?

Reply:
Alternatively, we could use an implementation like seen at https://github.com/ocsigen/js_of_ocaml/blob/master/runtime/fs.js#L59-L70 to avoid a full-parse on the file/glob path.